### ISSUE TITLE
Feature Request: Document ways to interact with RMINC objects

### ISSUE THREAD

**Issue by gdevenyi** [Mouse-Imaging-Centre/RMINC#214]
Recently ran into wanting to load a subregion from a bunch of volumes, I have a nice ROI defined with mask, it would be nice to directly use that to filter loading.

mincGetVolume("myfile.mnc", mask="myROI.mnc")

**Comment by cfhammill** [comment]
Hi Gabe,

Is this mostly a convenience issue? Or a speed issue?

```
mask <- mincGetVolume("myROI.mnc")
read_masked <- function(file, mask) mincGetVolume(file)[mask > .5]
```

Seems easy enough no?

**Comment by gdevenyi** [comment]
Initially a convenience option, for large numbers of files or large files one could conceive of wanting to avoid loading the whole file.

Once again my R newbness is exposed.

**Comment by cfhammill** [comment]
TLDR: API design is hard.

**Comment by nzxwang** [comment]
Although we talked in person, just for the record my opinion:

I like the idea of having a small function like `mincSubset( mincGetVolume_output, mask )`, and also improving our documentation with examples that point to these smaller functions. On a related note, maybe even another function like `mincMask()` that returns the entire mincSingleDim with 0's for masked out voxels.

**Comment by gdevenyi** [comment]
Thanks for the discussion @cfhammill 

This is pretty close to what I was thinking about with regards to R/RMINC internals.

I was thinking a bit of of documentation describing the "structure" of the various RMINC variable types and how you can manipulate them could be a long way to helping the non-R-wizards figure out some of this on their own.
